uint32 uid
double pos_x
double pos_y
double orientation

byte DEPLOYMENT_POINT = 1
byte WAYPOINT = 2

byte type # either DEPLOYMENT_POINT or WAYPOINT

